src/imiscsvrinfo.cpp

```diff
@@ -83,6 +85,18 @@ main( int argc, char **argv ) {
         hr = hr % 24;
         printf( "up %d days, %d:%d\n", day, hr, min );
     }
+    if(miscSvrInfo->certinfo.len > 0) {
+        const char* certinfobuf = static_cast<char*>(miscSvrInfo->certinfo.buf);
+        printf("SSL Info:\n");
```

Comment:
is it SSL or SSL/TLS or TLS?   hmmm

Reply:
I've been using "SSL/TLS" because TLS seems to be the favored term these days but we have an SSL plugin.

Reply:
SSL/TLS is the most accurate representation AFAIK. That's what I've been using as well.

However, I don't think it matters much whether it's SSL, SSL/TLS, or TLS as long as the user understands what's being communicated.

Reply:
Looks like @FifthPotato has settled on `SSL/TLS Info`.

Resolving this.